docs/census-open-data-registry.yaml

```diff
@@ -0,0 +1,104 @@
+Name: CZ CELLxGENE Discover Census
+Description: The Census provides efficient computational tooling to access, query, and analyze all single-cell RNA data from CZ CELLxGENE Discover.
```

Comment:
It provides the data too, not just the tooling. Not sure if the details matter here, but since this application is about sharing the _data_, seems like we should focus on that?

Reply:
This needs to be rewritten for sure, I just got it from the docsite. I think it should:
1. Be self-sufficient: assume readers don't know what cellxgene is or even what the data is about
2. Be longer. Descriptions from most other projects are a few paragraphs on average.
3. As you stated, be more descriptive about what type of data is contained.

I believe @pablo-gar is going to rewrite this, but contributions are welcome.

Reply:
```suggestion
Description: CZ CELLxGENE Discover is the largest repository of standardized single-cell gene expression data. It is considered by the scientific community to be the gold reference of human and mouse single-cell data.
This year CZ CELLxGENE Discover has made available all of its human and mouse RNA single-cell data through Census (https://chanzuckerberg.github.io/cellxgene-census/) – a free-to-use service with an API and data that allows for querying its single-cell data corpus directly from Python or R. 
The API uses a new technology, TileDB-SOMA, that allows for efficient and low-latency querying. The data are fully standardized and hosted publicly for free access, and they are composed by a count matrix of 33M unique cells (observations) by >60 k genes (features) accompanied by 11 cell metadata variables (e.g. cell type, tissue, sequencing technology, donor id, etc)  and gene metadata that includes GENCODE-based IDs and gene names. While these data are built from more than 500 datasets, the APIs enable convenient cell- and gene-based filtering to obtain any slice of interest in a matter of seconds. All data can be quickly transformed to NumPy, Pandas, Anndata, Seurat, or R base objects. We created data loaders for the data to be directly used by PyTorch for modeling at scale.
```

Reply:
For posterity, the above text was taken from our [initial sponsorship application](https://docs.google.com/document/d/1LhLExOakAN_4nVRsCRq7PbKWWrpPJZll66Nyk_MezYA/edit#heading=h.kvijxywrqloz).